azurefunctions-extensions-bindings-blob/azurefunctions/extensions/bindings/blob/utils.py

```diff
@@ -0,0 +1,26 @@
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License.
+import os
+
+
+def validate_connection_string(connection_string: str) -> str:
```

Comment:
[since extens](https://pypi.org/project/pydantic/) pydantic supports 3.8+ too, unsure if it is planned to be added soon, this can be a way to decouple domain logic from validating logic(auto type conversion/validation at runtime) and promote code reuse. i would propose something like
```
class ConnectionConfig(BaseModel):
    connection_string: str = Field(..., description="Storage account connection string")

    @validator("connection_string")
    def validate_connection_string(cls, v):
        if not os.getenv(v):
            raise ValueError(f"Storage account connection string {v} does not exist. Please make sure that it is a defined App Setting.")
        return v
```